samples/java/play-authenticate-usage/app/models/User.java

```diff
@@ -173,7 +172,7 @@ public static User create(final AuthUser authUser) {
 		}
 
 		user.save();
-		user.saveManyToManyAssociations("roles");
+		// user.saveManyToManyAssociations("roles");
```

Comment:
It seems so, the method `saveManyToManyAssociations` does not exist anymore. I was running the app and did some testing and it seems the role was saved correctly. E.g. I could not access the restricted demo page.